docs/quickstart/quickstart_chains/ethereum-uniswap.md

```diff
@@ -0,0 +1,264 @@
+
+::: warning
+We suggest starting with the [Ethereum Gravatar example](./ethereum-gravatar). The Uniswap project is a lot more complicated and introduces some more advanced concepts.
+:::
+
+Now, let's move forward and fork the example code for this project from [here](https://github.com/subquery/ethereum-subql-starter/tree/main/Ethereum/ethereum-uniswap-v3)
+
+## 1. Your Project Manifest File
+
+The Project Manifest (`project.yaml`) file works as an entry point to your Ethereum project. It defines most of the details on how SubQuery will index and transform the chain data. For Ethereum, there are three types of mapping handlers (and you can have more than one in each project):
+
+- [BlockHanders](../../build/manifest/ethereum.md#mapping-handlers-and-filters): On each and every block, run a mapping function
+- [TransactionHandlers](../../build/manifest/ethereum.md#mapping-handlers-and-filters): On each and every transaction that matches optional filter criteria, run a mapping function
+- [LogHanders](../../build/manifest/ethereum.md#mapping-handlers-and-filters): On each and every log that matches optional filter criteria, run a mapping function
+
+In this Uniswap project, the primary focus is on indexing logs exclusively from two Uniswap smart contracts. LogHandlers, which are the prevalent type of Ethereum handlers, are showcased in this example project, totaling 10 distinct log handlers.
+
+Additionally, it's important to highlight that this project incorporates six different ABIs. Furthermore, to accommodate scenarios where a contract factory generates new contract instances, [dynamic data sources](../../build/dynamicdatasources.md) are employed.
+
+```yaml
+dataSources:
+  - kind: ethereum/Runtime
+    startBlock: 12369621
+    options:
+      # Must be a key of assets
+      abi: Factory
+      address: "0x1F98431c8aD98523631AE4a59f267346ea31F984"
+    assets:
+      Factory:
+        file: "./abis/Factory.json"
+      ERC20:
+        file: "./abis/ERC20.json"
+      ERC20SymbolBytes:
+        file: "./abis/ERC20SymbolBytes.json"
+      ERC20NameBytes:
+        file: "./abis/ERC20NameBytes.json"
+      Pool:
+        file: "./abis/pool.json"
+    mapping: ...
+    # ethereum/contract
+  - kind: ethereum/Runtime
+    startBlock: 12369651
+    options:
+      abi: NonfungiblePositionManager
+      address: "0xC36442b4a4522E871399CD717aBDD847Ab11FE88"
+    assets:
+      NonfungiblePositionManager:
+        file: ./abis/NonfungiblePositionManager.json
+      Pool:
+        file: ./abis/pool.json
+      Factory:
+        file: ./abis/factory.json
+      ERC20:
+        file: ./abis/ERC20.json
+    mapping: ...
+templates:
+  - name: Pool
+    kind: ethereum/Runtime
+    options:
+      abi: Pool
+    assets:
+      Pool:
+        file: "./abis/pool.json"
+      Factory:
+        file: "./abis/factory.json"
+      ERC20:
+        file: "./abis/ERC20.json"
+    mapping: ...
+```
+
```

Comment:
```suggestion
Check out our [Manifest File](https://github.com/subquery/documentation/blob/master/docs/build/manifest/ethereum.md) documentation to get more information about the Project Manifest (project.yaml) file.
```

Reply:
I think it would be more beneficial to use a link to a local page instead of one that directs to markdown hosted on Github. This is the consistent formatting used in all instances.

Reply:
Ok, please adjust